app/helpers/application_helper.rb

```diff
@@ -144,7 +144,10 @@ def cancel_button(cancel_path = :back)
     link_to t('cancel'), cancel_path, method: :get, class: 'btn btn-primary formSubmitr', data: {:confirm =>  t('cancel_confirm')}
   end
 
-  def link_to_if_i_can label, url, object:, permission: :show
+  def link_to_if_i_can label, url, object: nil, permission: :show
```

Comment:
Sur la page http://localhost:3000/workgroups/1/imports/3 le helper link_to_if_can_show n'est pas chargé : 

~~~sh
ActionView::Template::Error (undefined method `link_to_if_can_show' for #<#<Class:0x00007fc04134d718>:0x00007fc05e2c6890>
Did you mean?  link_to_if_i_can):
    24:           key: :status, \
    25:           attribute: lambda do |n| \
    26:             url = n.child_import.present? ? [parent, n.child_import] : [parent, @import, n]; \
    27:             link_to_if_can_show(url.last, operation_status(n.child_import&.status || n.status, verbose: true, default_status: :pending), url) \
    28:           end, \
    29:           sortable: false, \
    30:         ), *controls \
  app/views/imports/import/_workbench.html.slim:27:in `block in _app_views_imports_import__workbench_html_slim__2062437056068468558_70231849675400'
  app/helpers/table_builder_helper/column.rb:22:in `value'
  app/helpers/table_builder_helper.rb:233:in `block (2 levels) in tr'
  app/helpers/table_builder_helper.rb:232:in `each'
  app/helpers/table_builder_helper.rb:232:in `block in tr'
  app/helpers/table_builder_helper.rb:222:in `tr'
  app/helpers/table_builder_helper.rb:318:in `block (2 levels) in tbody'
  app/helpers/table_builder_helper.rb:317:in `block in tbody'
  app/helpers/table_builder_helper.rb:316:in `tbody'
  app/helpers/table_builder_helper.rb:97:in `table_builder_2'
  app/views/imports/import/_workbench.html.slim:14:in `_app_views_imports_import__workbench_html_slim__2062437056068468558_70231849675400'
  app/views/imports/show.html.slim:6:in `_app_views_imports_show_html_slim___3247989105483851095_70231851896440'
  app/controllers/concerns/iev_interfaces.rb:13:in `show'
  config/middlewares/cachesettings.rb:8:in `call'
~~~